LoopKitUI/Views/NewScheduleItemEditor.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  NewScheduleItemEditor.swift
+//  LoopKitUI
+//
+//  Created by Michael Pangburn on 5/7/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import SwiftUI
+import LoopKit
+
+
+struct NewScheduleItemEditor<Value, ValuePicker: View>: View {
+    @Binding var isPresented: Bool
+    @State var item: RepeatingScheduleValue<Value>
+    var initialItem: RepeatingScheduleValue<Value>
+    var unavailableTimes: Set<TimeInterval>
+    var valuePicker: (_ item: Binding<RepeatingScheduleValue<Value>>) -> ValuePicker
+    var save: (RepeatingScheduleValue<Value>) -> Void
+
+    init(
+        isPresented: Binding<Bool>,
+        initialItem: RepeatingScheduleValue<Value>,
+        unavailableTimes: Set<TimeInterval>,
+        @ViewBuilder valuePicker: @escaping (_ item: Binding<RepeatingScheduleValue<Value>>) -> ValuePicker,
+        onSave save: @escaping (RepeatingScheduleValue<Value>) -> Void
+    ) {
+        self._isPresented = isPresented
+        self._item = State(initialValue: initialItem)
+        self.initialItem = initialItem
+        self.unavailableTimes = unavailableTimes
+        self.valuePicker = valuePicker
+        self.save = save
+    }
+
+    var body: some View {
+        VStack(spacing: 0) {
+            PseudoNavigationBar(
+                leading: { cancelButton },
+                center: {
+                    Text("New Entry")
+                        .font(.headline)
+                },
+                trailing: { addButton }
```

Comment:
What is a `PseudoNavigationBar` and why is it needed?

Reply:
Pete raised the same (good) question; my response is above.
I've renamed to NavigationViewHeader, as the use of "pseudo" is awkward.

Reply:
How about something not as strongly implying some relationship existing view types, as that's what threw me off.  If it was just something like "ModalHeaderButtonBar" or something that would work and wouldn't look like you're trying to reimplement or conform to an pattern.

Reply:
ModalHeaderButtonBar works for me.